backend/src/test/java/com/huellapositiva/integration/AwsEmailServiceShould.java

```diff
@@ -18,7 +18,7 @@
 @ExtendWith(AwsEnvVariablesExtension.class)
 @ExtendWith(LocalstackDockerExtension.class)
 @LocalstackDockerProperties(services = { "ses" })
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE, properties = {"huellapositiva.feature.email.enabled=true"})
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT, properties = {"huellapositiva.feature.email.enabled=true"})
```

Comment:
Why is that necessary?

Reply:
I was having some issues related with AWS Service Email Test so I figure out why this test has not a defined port, when I change this property with the rest of Cors config enabled this test works again. 

I'm not sure at all why but this change makes test passed with cors config enabled

Reply:
I found this when executing webEnvironment to .NONE throws an exception saying that Its necesary to define this **Consider defining a bean named mvcHandlerMappingIntrospector in your configuration.**
 So that was the reason of changing to DEFINED_PORT

Reply:
Miraremos eso en el futuro